tests/test_iframe.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+""""
+Folium Element Module class IFrame
+----------------------
+"""
+import branca.element as elem
+from selenium import webdriver
+import base64
```

Comment:
There are a few coding standards issues here. From the Travis-CI logs:

```
./tests/test_iframe.py:8:1: F401 'base64' imported but unused
./tests/test_iframe.py:9:1: E302 expected 2 blank lines, found 0
./tests/test_iframe.py:10:1: W191 indentation contains tabs
./tests/test_iframe.py:11:1: W191 indentation contains tabs
./tests/test_iframe.py:13:1: E302 expected 2 blank lines, found 1
./tests/test_iframe.py:14:1: W191 indentation contains tabs
./tests/test_iframe.py:14:48: E231 missing whitespace after ','
./tests/test_iframe.py:14:57: E231 missing whitespace after ','
./tests/test_iframe.py:15:1: W191 indentation contains tabs
./tests/test_iframe.py:17:1: E302 expected 2 blank lines, found 1
./tests/test_iframe.py:18:1: W191 indentation contains tabs
./tests/test_iframe.py:19:1: W191 indentation contains tabs
./tests/test_iframe.py:20:1: W191 indentation contains tabs
./tests/test_iframe.py:21:1: W191 indentation contains tabs
./tests/test_iframe.py:21:27: E703 statement ends with a semicolon
./tests/test_iframe.py:22:1: W191 indentation contains tabs
```

Those should be fairly easy to fix. Thanks for you patience @knil-sama! We are almost ready to merge this!